.gitignore

```diff
@@ -1,8 +1,3 @@
-# Dynamically generated by IntelliJ IDEA FRC Plugin on Friday October 20, 2023 9:53:51 AM PDT
-# using https://www.toptal.com/developers/gitignore/api (See https://docs.gitignore.io)
-# Created by https://www.toptal.com/developers/gitignore/api/java,gradle,intellij+all,visualstudiocode,eclipse,netbeans,linux,macos,windows
-# Edit at https://www.toptal.com/developers/gitignore?templates=java,gradle,intellij+all,visualstudiocode,eclipse,netbeans,linux,macos,windows
```

Comment:
why are you removing the IntelliJ banner

Reply:
it was a little annoying